salt/utils/files.py

```diff
@@ -904,3 +904,18 @@ def check_system_encoding(_data):
         return "ASCII"
 
     raise CommandExecutionError("Could not detect file encoding")
+
+
+def get_machine_identifier():
```

Comment:
Why is this is `salt.utils.files` and not in `salt.utils.platform`?

Reply:
Added it to salt files since it was doing file access to get the machine identifier.  Adding to salt.utils.platform not against, just files seemed more convenient and correct since it was doing file access.

Reply:
The information collected, though, from files, is platform related, the machine ID

Reply:
agree with @s0undt3ch about this. `salt.utils.files` should be reserved for methods to access files. while `salt.utils.platform` is reserved for how to determine information about the platform being accessed. regardless of methods used to do such. 

as such it is about what it does not how it does it.

Reply:
okay moved it to platform, testing that now